### DISCUSSION TITLE
Access corporate GitHub server with the mobile app

### DISCUSSION THREAD

**Discussion by ilyas409** [community/community#3296] [Enterprise]
I use GitHub Desktop on my work machine connecting to our corporate repo.
However, I'm not able to do that on mobile.
Currently there's no option to log in into a corporate account using the mobile app

**Comment by candyho** [comment]
Hey there, we have just shipped a Beta for GitHub Enterprise Server 3.0 support on mobile recently. You need to be on GitHub Enterprise Server instance version 3.0 or greater, and install the latest version of our apps on your device . If you have met these requirements, you could sign in with a user account on GitHub.com, then long-press your Profile avatar at the bottom to add your Enterprise Account. Hope this helps!

You can find more information about GitHub Enterprise Server 3.0 Support on mobile here - https://docs.github.com/en/enterprise-server@3.0/github/getting-started-with-github/github-for-mobile
